src/Components/ReviewListing.jsx

```diff
@@ -0,0 +1,11 @@
+export default function ReviewListing({title, designer, owner, review_img_url, category, created_at, votes, comment_count}) {
+    return (
+        <article id="reviewList">
+            <ul className="card">
+                <li><img src={review_img_url} alt="none found" id="listing"/></li>
+                <li>{title}</li>
```

Comment:
I would probably change the html of this, \<li> tags are meant to house other tags, {title} is going to be a piece of text and therefore should be wrapped in a p tag. Also it doesnt make much sense as these items aren't really part of a list, they are just elements and you are using a list to render. You should be using CSS to style these correctly over html tags.

Reply:
I have removed the list tags too.